.vscode/tasks.json

```diff
@@ -1,23 +1,22 @@
 {
-	"version": "2.0.0",
-	"tasks": [
-		{
-			"type": "npm",
-			"script": "compile",
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			},
-			"isBackground": true,
-			"presentation": {
-				"reveal": "silent"
-			},
-			"problemMatcher": "$tsc-watch"
-		},
-		{
-			"type": "npm",
-			"script": "lint",
-			"problemMatcher": "$tslint5"
-		}
-	]
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "npm",
+            "script": "compile",
+            "problemMatcher": [
```

Comment:
Can you leave the "group" and "presentation" properties in? I will sometimes call `Run Build Task` from the command palette to start the watch task and those result in a few less prompts